File: app/models/servico.py
from datetime import datetime
from ..utils import gerar_id

def criar_servico(tipo, descricao, data, hora, cliente_id, pet_id):
    return {
        'id': gerar_id(),
        'tipo': tipo,
        'descricao': descricao,
        'data': data,
        'hora': hora,
        'cliente_id': cliente_id,
        'pet_id': pet_id,
        'prestador_id': None,
        'status': 'pendente',
        'criado_em': datetime.utcnow(),
        'atualizado_em': datetime.utcnow()
    }

File: woofit_cliente/lib/models/servico.dart
class Servico {
  final String id;
  final String tipo;
  final String descricao;
  final String data;
  final String hora;
  final String clienteId;
  final String petId;
  final String? prestadorId;
  final String status;

  Servico({
    required this.id,
    required this.tipo,
    required this.descricao,
    required this.data,
    required this.hora,
    required this.clienteId,
    required this.petId,
    this.prestadorId,
    required this.status,
  });

  factory Servico.fromJson(Map<String, dynamic> json) => Servico(
        id: json['id'],
        tipo: json['tipo'],
        descricao: json['descricao'] ?? '',
        data: json['data'],
        hora: json['hora'],
        clienteId: json['cliente_id'],
        petId: json['pet_id'],
        prestadorId: json['prestador_id'],
        status: json['status'],
      );
}